setup.py

```diff
@@ -16,39 +16,53 @@
 
 import os
 import sys
+import glob
+import platform
+import shutil
 from setuptools import find_packages
 from setuptools import setup, Extension
 from setuptools.command.install import install
 from setuptools.command.build_ext import build_ext
 
+VERSION = '2.10b3'
 
 class CMakeExtension(Extension):
 
   def __init__(self, name):
     # don't invoke the original build_ext for this special extension
-    super().__init__(name, sources=[])
+    # setuptools is having troubles to build empty list of sources on Windows
+    sources = ['third_party/gfootball_engine/src/misc/empty.cpp'] if platform.system() == 'Windows' else []
+    super().__init__(name, sources=sources, optional=True)
```

Comment:
Setuptools raises an exception on Windows if the list of sources is empty. Optional argument helps to proceed installation, but it fails in development (editable) mode (pip install -e .). That's why an empty cpp file is necessary to proceed.

Reply:
Lets just add empty.cpp unconditionally then